src/legacy/core_plugins/kbn_vislib_vis_types/public/gauge.d.ts

```diff
@@ -17,37 +17,39 @@
  * under the License.
  */
 
-import { ColorSchemas } from 'ui/vislib/components/color/colormaps';
+import { RangeValues } from 'ui/vis/editors/default/controls/ranges';
+import { Alignments, GaugeTypes } from './utils/collections';
+import { ColorSchemaVislibParams } from './types';
+
+interface Gauge extends ColorSchemaVislibParams {
+  backStyle: 'Full';
+  gaugeStyle: 'Full';
+  orientation: 'vertical';
+  type: 'meter';
+  alignment: Alignments;
+  colorsRange: RangeValues[];
+  extendRange: boolean;
+  gaugeType: GaugeTypes;
+  labels: {
+    show: boolean;
+  };
+  minAngle?: number;
+  maxAngle?: number;
```

Comment:
Since we got rid of `maxAngle` and `minAngle` from the config, let's remove it definition as well.

Reply:
Right, removed